homeassistant/components/totalconnect/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: todo
+  test-before-setup: todo
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Total Connect entities do not subscribe to events.
+  dependency-transparency: done
+  action-setup: todo
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  docs-actions: done
+  brands: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: todo
+  entity-unavailable: todo
+  action-exceptions: todo
+  reauthentication-flow: done
+  parallel-updates: todo
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters: done
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery:
+    status: exempt
+    comment: |
+      Total Connect is not discoverable
```

Comment:
Did you try if the Mac address of the device is registered as OUI?

Reply:
I will change it to `todo` for now and look into the OUI lookup.